app/store/index.js

```diff
@@ -50,8 +50,8 @@ const store = new Vuex.Store({
     async REQUEST_DATA ({ commit }) {
       try {
         commit('HIDE_TRANSLATION')
-        const response = await axios.get(`${TRANSLATIONS_ENDPOINT}?lang=${store.state.language}`)
-        const translation = sample(response.data)
+        const response = await axios.get(`${TRANSLATIONS_ENDPOINT}?language_code=${store.state.language}`)
+        const translation = sample(response.data.data.attributes)
```

Comment:
Is the API adding an extra `data` key to the response?

Reply:
@rkillackey  Yeah, i added active model serializer and rails__util so the request's response (which has data nested under response.data) is now `{ data: { type: '', attributes: [] } }`,

Reply:
Gotcha. We haven't deploy the API update to production have we? Should we wait to do this until maybe around 5 today, so less people are hopefully using it at the time?

Reply:
^ Nope, I didn't want to break it so we should def do it at the same time to minimize conflicts for our vast pool of users haha. 5 sounds good.

Reply:
Hahaha good point...our users are few but loyal